src/utils.py

```diff
@@ -275,3 +297,103 @@ def upload_experiment_to_s3(experiment_id: str,
     for upload_file in tqdm(upload_files):
         upload_file = str(upload_file)
         s3_client.upload_file(upload_file, bucket_name, upload_file.replace(current_global, object_global))
+
+
+def raven_to_csv_annotations(annotations_path: str,
+                             recording_path: str,
+                             dataset_name: str = 'recordings_2018_filtered'):
+
+    """
+    Takes annotation files (selection table) created in Raven and turns it to a compatible annotations csv.
+    """
+    #create dataframe
+    annotations=Path(annotations_path)
+    recording=Path(recording_path)
+    filelist = list(annotations.glob('*selections.txt')) #ignore irrelevent files
+    metadata = []
+    for file in filelist:
+        dfTemp = pd.read_csv(file, sep="\t")
+        dfTemp['filename'] = re.search(r'18\d{4,4}_\d{6,6}', file.as_posix()).group()
```

Comment:
Is it specific for our files? Because this looks specific

Reply:
you're right, I'll generalize it

Reply:
I also won't work for 2019 tags. 
I think that maybe separating each row by tabs/spaces will return the needed fields

Reply:
worked on it today. changed the search to ".selection" so it's generalized to any raven file.
I thought about having it look for ".txt" to be even more generalized, and then instruct the user to not have any other text files in the folder, but we first need to do it ourselves (i.e. we have txt files in our finding_willy/annotations, so it wouldn't have worked for you guys had done it).